### PR TITLE
set first sample of output in Ctor

### PR DESCRIPTION
**Pull request by sonoro1234**
possibly solves #214

Also need that in DWGBowed , DWGPlucked and OteyPianoUGens. Shall I wait for this commited or include changes in this PR?

### CODE REVIEW

**Comment by jamshark70** [comment]
I'd say, include those fixes here. If the issue and solution are known, there's no need for a separate PR.

Thanks!

File: source/StkInst/StkInst.cpp [hunk 1]
```diff
@@ -411,6 +412,7 @@ void StkGlobals_Ctor(StkGlobals* unit) {
     Stk::printErrors(unit->printErrors);
     RTFree(unit->mWorld,unit->rawwavePath);
     SETCALC(StkGlobals_next);
+    (unit->mCalcFunc)(unit, 1);
```
Comment (snappizz) [inline source/StkInst/StkInst.cpp]: why call calcfunc in the Dtor?

**Comment by snappizz** [comment]
(deleted a comment which was based on a misunderstanding)

**Comment by sonoro1234** [comment]
done

**Comment by sonoro1234** [comment]
Is somebody going to review?

**Comment by sonoro1234** [comment]
3.10.0-rc1?

File: source/OteyPianoUGens/OteyPiano.cpp [hunk 2]
```diff
@@ -328,6 +329,7 @@ void OteyPianoStrings_Ctor(OteyPianoStrings* unit) {
 	new(unit) OteyPianoStrings(unit);
 	unit->piano.init(freq,SAMPLERATE,velocity*10,minr,maxr,amprl,amprr,rcore,minl,maxl,ampll,amplr,rho,young,zb,zh,mh,k,alpha,p,ph,loss,detune,hammer_type);
 	SETCALC(OteyPianoStrings_next);
+	ZOUT0(0) = 0.0;
```
Comment (snappizz) [inline source/OteyPianoUGens/OteyPiano.cpp]: `ZOUT0` and all other `Z*` macros are legacy. please use `ClearUnitOutputs(unit, 1)` here and elsewhere.

**Comment by sonoro1234** [comment]
Done

**Pull request merged**